package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/digitalbazaar/eslint-config-digitalbazaar#readme",
   "dependencies": {
-    "eslint": "^5.13.0"
+    "eslint": "^5.13.0",
```

Comment:
since we decided to install these as `devDependencies` in all the places then I think they would be best be `optionalDependencies` here.  Aren't we missing something here related to Vue anyhow?

Reply:
yeah I can remove the plugin.

Reply:
actually hold @dlongley I believe you wanted eslint as a dependency and then eslint-config-digitalbazaar installed as a dev dependency. as the plugin for js-doc it can be removed or move to optional dependency. I will look into it.